test/common.sh

```diff
@@ -72,20 +76,28 @@ function deploy() {
 		--tenant "${TENANT_ID}" &>/dev/null
 
 	az account set --subscription "${SUBSCRIPTION_ID}"
+}
+
+function deploy_template() {
+	# Check pre-requisites
+	[[ ! -z "${DEPLOYMENT_NAME:-}" ]] || (echo "Must specify DEPLOYMENT_NAME" && exit -1)
```

Comment:
Can we consolidate on `INSTANCE_NAME` or `DEPLOYMENT_NAME`? For 99.999% of cases, they're likely to be the same, no?